config/filter.d/postfix-rbl.conf

```diff
@@ -13,6 +13,7 @@ before = common.conf
 _daemon = postfix(-\w+)?/smtpd
 
 failregex = ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 454 4\.7\.1 Service unavailable; Client host \[\S+\] blocked using .* from=<\S*> to=<\S+> proto=ESMTP helo=<\S*>$
+            ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 554 5\.7\.1 Service unavailable; Client host \[\S+\] blocked using .* from=<\S*> to=<\S+> proto=SMTP helo=<\S*>$
```

Comment:
Sure: I started following https://github.com/fail2ban/fail2ban/blob/0.10/config/filter.d/postfix.conf example, but I was waiting for a feedback like yours.
Tell me if this looks ok